src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -828,11 +839,47 @@ public void SarifTransformerTests_ToCurrent_NotificationExceptionWithStack()
                   },
                   ""frames"": [
                     {
+                      ""location"": {
+                        ""physicalLocation"": {
+                          ""fileLocation"": {
+                            ""uri"": ""file:///C:/src/main.cs""
+                          },
+                          ""region"": {
+                            ""startLine"": 15,
+                            ""startColumn"": 9
+                          }
+                        },
+                        ""fullyQualifiedLogicalName"": ""Rules.SecureHashAlgorithmRule.Evaluate-0"",
+                        ""message"": {
+                          ""text"": ""Exception thrown""
+                        }
+                      },
                       ""module"": ""RuleLibrary"",
                       ""threadId"": 52,
                       ""address"": 10092852
                     },
                     {
+                      ""location"": {
+                        ""physicalLocation"": {
+                          ""fileLocation"": {
+                            ""uri"": ""file:///C:/src/main.cs""
+                          }
+                        },
+                        ""fullyQualifiedLogicalName"": ""Rules.SecureHashAlgorithmRule.Register""
```

Comment:
Whereas this one it decided _not_ to disambiguate. Perhaps it's the `kind` on `Evaluate` that didn't match. If the transformer knows what `kind` is, though, it should populate it. We'll see...